test/models/chart_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```

Comment:
We'll be using RSpec to test, so this whole `test` folder generally will be deleted.

Reply:
This isn't actionable though, so don't worry about it for now.